fluent-package/apt/confluent-test.sh

```diff
@@ -34,41 +34,15 @@ chmod 644 /usr/share/keyrings/confluent-archive-keyring.gpg
 echo "deb [arch=${architecture} signed-by=/usr/share/keyrings/confluent-archive-keyring.gpg] https://packages.confluent.io/deb/7.6 stable main" > /etc/apt/sources.list.d/confluent.list
 apt update && apt install -y confluent-community-2.13 ${java_jdk} netcat-openbsd
 
-export KAFKA_OPTS=-Dzookeeper.4lw.commands.whitelist=ruok
-/usr/bin/zookeeper-server-start /etc/kafka/zookeeper.properties  &
-N_POLLING=30
-n=1
-while true ; do
-    sleep 1
-    status=$(echo ruok | nc localhost 2181)
-    if [ "$status" = "imok" ]; then
-	break
-    fi
-    n=$((n + 1))
-    if [ $n -ge $N_POLLING ]; then
-	echo "failed to get response from zookeeper-server"
-	exit 1
-    fi
-done
-/usr/bin/kafka-server-start /etc/kafka/server.properties &
-n=1
-while true ; do
-    sleep 1
-    status=$(/usr/bin/zookeeper-shell localhost:2181 ls /brokers/ids | sed -n 6p)
-    if [ "$status" = "[0]" ]; then
-	break
-    fi
-    n=$((n + 1))
-    if [ $n -ge $N_POLLING ]; then
-	echo "failed to get response from kafka-server"
-	exit 1
-    fi
-done
+CONFLUENT_SCRIPT=$(dirname $(realpath $0))/../run-confluent.sh
+echo ${CONFLUENT_SCRIPT}
+bash ${CONFLUENT_SCRIPT}
+
 if [ "$CI" = "true" ]; then
    echo "::endgroup::"
 fi
-/usr/bin/kafka-topics --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic test
+
 export PATH=/opt/fluent/bin:$PATH
 export INSTALLATION_TEST=true
-cd /fluentd && rake serverspec:kafka
 /usr/sbin/fluentd -c /fluentd/serverspec/test.conf &
+cd /fluentd && rake serverspec:kafka
```

Comment:
Could you tell me why the order was changed?

Reply:
make it consistent with apt/confluent-test.sh.